pkgs/development/python-modules/gradio/default.nix

```diff
@@ -79,55 +78,52 @@ buildPythonPackage rec {
   ];
 
   propagatedBuildInputs = [
-    setuptools # needs pkg_resources
+    setuptools # needed for 'pkg_resources'
```

Comment:
should we keep dependency order the same as in the requirement.txt , just to make it easier to spot updates ?
https://github.com/gradio-app/gradio/blob/main/requirements.txt

Reply:
Aside from setuptools it should be that same?

Reply:
right , I had only looked before the change. Your change fixed this.

Reply:
They sorted it upstream, so I followed suit